frontend/src/components/Rides/Ride.js

```diff
@@ -15,7 +15,7 @@ class Ride extends Component {
         }
     }
 
-    componentDidMount() {
+    componentWillMount() {
```

Comment:
I needed to have the driver info ready before the component was rendered, since the componentWillMount() calls getUserInfo(), which makes an API call as the component loads. I think if I didn't do this the driver info wasn't displaying properly